### PR TITLE
Add 'Home' and 'About' pages version mobile & desktop

### PR DESCRIPTION
**Pull request by Nejjari001**
- Build Responsive site from Mobile to desktop.
- This site will be a replica of CC-Global-Summit-2015 website from **Behance** gallery. 
- It consists of two pages (**Home** & **About** pages) 
- This is the link of the recorded video presenting the features of the site: https://www.loom.com/share/78bea9b6b78c42c19f6740966f74c8b6

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,1010 @@
...
+.menu-list-mobile {
+  position: fixed;
+  display: none;
+  background: var(--color2);
+  opacity: 1;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+}
```
Comment (Chimwemwe-127001) [inline css/styles.css]: - [ ] It appears that the images including the logo in the content area are overlapping with the mobile navigation menu. Consider fixing this small issue.

about section mobile

File: about.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+            </p>
+            <div class="contact">
+              <p class="contact-text center">
+                Please contact us per Email for any further questions about the
+                Arab Affiliate Morocco Nov 2022
+              </p>
+              <p class="contact-email center">arabafiliatesummit@sumit2022.org</p>
+            </div>
+          </div>
+      </header>
```
Comment (Chimwemwe-127001) [inline about.html:88]: - [ ] It appears that the images including the logo in the content area are overlapping with the desktop navigation bar. Consider fixing this small issue.

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
### Hi @Nejjari001 

Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlight :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:

## Required Changes ♻️
- [ ] It is imperative that you record a video presenting the features of the project you built. It is a video with a **maximum length of 5 minutes**. For more insight about the video presentation check the the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) under the `Project documentaion` section.
- [ ] The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must add appropriate credit to the author in the `Acknowledgement` section of your README file.
- [ ] Consider making use of GitHub Pages to deploy the web pages and add a link to the deployed project in your README file.

***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by alexander16108 — APPROVED** [review]
Hi @Nejjari001, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Clean code ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**